third_party/scv-tr/src/scv-tr/scv_tr_text.cpp

```diff
@@ -321,8 +321,8 @@ static void do_attributes(bool declare_attributes, // If false then print the va
         }
     } break;
     default: {
-        std::string s = fmt::format(FMT_STRING("Unsupported attribute type = {0}."), my_exts_p->get_type());
-        _scv_message::message(_scv_message::TRANSACTION_RECORDING_INTERNAL, s.c_str());
+        // std::string s = fmt::format(FMT_STRING("Unsupported attribute type = {0}."), my_exts_p->get_type());
```

Comment:
commenting out is a bad solution. Issue here is that fmt cannot convert the enum returned by get_type(). A static_cast<unsigned>() solves the issue. Pls. change it accorsingly.